### PR TITLE
ci: add default linter action

### PR DESCRIPTION
**Pull request by kweinmeister**
This PR adds a default linting action for multiple languages using Github's [super-linter](https://github.com/github/super-linter) tool.

### CODE REVIEW

File: .github/workflows/linter.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+      ################################
+      # Run Linter against code base #
+      ################################
+      - name: Lint Code Base
+        uses: github/super-linter@v4
+        env:
+          VALIDATE_ALL_CODEBASE: false
+          DEFAULT_BRANCH: main
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (nicain) [inline .github/workflows/linter.yaml:55]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

**Comment by kweinmeister** [comment]
> The "linting" job in dialogflow-cx-webhook-python.yaml is redundant, and can be removed in favor of this. Can a dependency be added across from this workflow (linter.yaml) to the other workflow (dialogflow-cx-webhook-python.yaml)? That way, we don't burn up CPU running the test suite until the linting is successful. (this is the way linting is structured in dialogflow-cx-webhook-python.yaml).

I have removed the linting job from the `dialogflow-cx-webhook-python.yaml` workflow. The pattern match on the webhook code will trigger both workflows. So, this workflow will handle linting, and the other one will hand the unit tests.

File: .github/workflows/linter.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+# https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions
+#
+
+#############################
+# Start the job on all push #
+#############################
+on:
+  push:
+    branches-ignore: [main]
+    # Remove the line above to run when pushing to master
```
Comment (nicain) [inline .github/workflows/linter.yaml:20]:
```suggestion
    # Remove the line above to run when pushing to main
```

**Review by nicain — COMMENTED** [review]
LGTM

**Pull request merged**